### PR TITLE
image templates for branching

### PR DESCRIPTION
**Pull request by hennevogel**

### CODE REVIEW

**Comment by mschnitzer** [comment]
One thing what came into my mind: Did we create a branching link?

**Comment by hennevogel** [comment]
@mschnitzer no because this is not part of this sprint. there is a card in the backlog

**Comment by adrianschroeter** [comment]
The attribute commit lacks a migration defining the attribute.

But I would rename it, since it seems to be more used as a collector marker (like OBS:Maintained) and not as change in the view (like the project meta "kind" attribute).

What About OBS:DefaultTemplates  ? 
There is no reason why it shouldn't work with non images, no?

**Comment by mschnitzer** [comment]
@hennevogel ok, so I suppose we should make the maintainers of the IBS and OBS aware that they should not update the instances (IBS & OBS) to the current git master because otherwise the view would be displayed to users and they wonder what to do with it (since there is no button and so on). Or we just don't merge it until we have both things done.

**Comment by adrianschroeter** [comment]
I don't see a test that the created repository gets also the rebuild="local" attribute, did I miss it?

**Comment by hennevogel** [comment]
@adrianschroeter attributes are defined in seeds. This is done.

**Comment by ChrisBr** [comment]
@adrianschroeter rebuild is already an option so we didn't implement it. However, as auto cleanup and publish is automatically set, maybe we should this option also set automatically ...

**Comment by adrianschroeter** [comment]
So far we did not document to re-seed on update, only db:migrate was documented and we created the attributes so far also via own migrations. So our admins will fail over this, I think.

We need to document this at least in README.UPDATERS. 
Or maybe we can hook in the re-seeding in the usual db:migrate run?

**Comment by hennevogel** [comment]
@adrianschroeter the rebuild policy was a new acceptance criteria, we ignored it :-)

**Comment by ChrisBr** [comment]
>  Or maybe we can hook in the re-seeding in the usual db:migrate run?
Please not!
Either update the docs README.UPDATERS or just add the migration. Don't make easy things more complex...

**Comment by bgeuken** [comment]
> > @hennevogel One thing what came into my mind: Did we create a branching link?

> @mschnitzer no because this is not part of this sprint. there is a card in the backlog

For that you would also miss the webui controller action.

File: src/api/Gemfile.lock [hunk 1]
```diff
@@ -400,3 +400,5 @@ DEPENDENCIES
   xmlhash (>= 1.3.6)
   yajl-ruby
 
+BUNDLED WITH
```
Comment (Ana06) [inline src/api/Gemfile.lock]: This is here again!!! :scream_cat:

**Comment by mschnitzer** [comment]
Fixed by https://github.com/openSUSE/open-build-service/pull/2349/commits/28f6875f3f03ef8fd0aee20bc1862e21a7cde4bd#diff-d275a16e04ad75fbabc5ef89e2cd6314R403

**Comment by hennevogel** [comment]
[(8) Present kiwi templates](https://trello.com/c/ti4SLGHR/93-8-present-kiwi-templates)

**Comment by hennevogel** [comment]
[(5) Branching image templates (api)](https://trello.com/c/qZRROUHe/96-5-branching-image-templates-api)

**Comment by hennevogel** [comment]
We are going to supersede this again...

**Pull request closed without merging**